llvm/docs/LangRef.rst

```diff
@@ -5336,6 +5336,7 @@ X86:
   operand in a SSE register. If AVX is also enabled, can also be a 256-bit
   vector operand in an AVX register. If AVX-512 is also enabled, can also be a
   512-bit vector operand in an AVX512 register. Otherwise, an error.
+- ``Ws``: A symbolic reference or label reference.
```

Comment:
What's the reason to choose `Ws`?

Reply:
x86 has almost run out of uppercase and lowercase letters... The remaining ones are `z` and `H`.

I tried `z` but one GCC x86 maintainer considers `W` prefix better: https://gcc.gnu.org/pipermail/gcc-patches/2024-January/642596.html

Reply:
Got it, thanks for the link! Do we support the pointer offset like GCC `&var + 1`?

Reply:
Yes, it's supported (aarch64 "S" supports it but riscv "S" doesn't) I intend to support it for x86 "Ws".

OK, I've figured out how to do it in SelectionDAG... Updated.